_posts/2017-01-22-Front-end-JavaScript-at-Artsy-2017.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Exploration: Front-end JavaScript at Artsy in 2017"
+date: 2017-01-22 12:17
+author: orta
+categories: [javascript, emission, reactnative, force, typescript]
+series: React Native at Artsy
+---
+
+The Artsy web team have been early adopters of node, and for the last 4 years the stable stack for the Artsy website has been predominantly been Node + CoffeeScript + Express + Backbone. In 2016 the mobile team announced that it had moved to React Native, matching the web team as using JavaScript as the tools of their trade.
+
+Historically we have always had two separate dev teams for building Artsy.net and the corresponding iOS app, we call them Collector Web, and Collector Mobile. By the end of 2016 we decided to merge the teams. The merger has given way to a whole plethora of ideas about what modern JavaScript looks like and we've been experimenting with finding common patterns between web and native.
```

Comment:
> Collector Web, and Collector Mobile

Might be useful describing the source/context of these terms, or "(Art) Collector Web, and Collector Mobile" or something similar.